### ISSUE TITLE
Send button's placeholder

### ISSUE THREAD

**Issue by slkwd** [Predictable-Dialogs/agent-embed#1]
The SEND button's placeholder, “_Type your answer_,” is not very intuitive in my opinion.
Is it possible to change it in the same way you do with the **Initial Response** in  SETTINGS? 

Thanks

**Comment by Jaikant** [comment]
@slkwd yes this text is sent by the backend, so it would have to be customized on the backend.
